core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2474,7 +2473,8 @@ private void loadMetadataAndJournal(LockedInodePath inodePath, LoadMetadataOptio
         }
       }
     } catch (IOException e) {
-      LOG.error(ExceptionUtils.getStackTrace(e));
+      LOG.error("Failed to loadMetadataAndJournal: inodePath={}, options={}.", inodePath.getUri(),
```

Comment:
usually we either log or throw, but not both since that leads to logging the same error multiple times

Reply:
Yes, agree. I did see this error logged multiple times from some master logs. 
I will remove it.

Reply:
actually it can be expected to reach here from `loadMetadataIfNotExistAndJournal`, therefore I think it makes sense to remove this logging